### DISCUSSION TITLE
Logback custom appender to push logs to ElasticSearch via Open Telemetry

### DISCUSSION THREAD

**Discussion by hegdemahendra** [open-telemetry/opentelemetry-java#3799] [Q&A] [answered]
Hello Team,

I am using apache Nifi , which uses logback framework for logging.
I have written a custom log appender to push logs to MongoDB.
Now I have a requirement to push logs in specific format (json) to ElasticSearch.

As few other apps are already using open telemetry to push data to ES, i have been asked to do so.

So now I am looking for right way to push data from my custom log appender (java class) to Open Telemetry.

Could some please point me to the usefull example which I can refer ?

Thanks
Mahendra

**Comment by jkwatson** [comment, marked as answer]
Hi @hegdemahendra . Our support for logs is *very* early days still. You are welcome to use the SDK logs module to try to do this. Your feedback is very welcome. Here's a PR in the instrumentation repo to do this for log4j: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/4375 Hopefully that will be useful to get to pointed in the right direction.
